packages/vue/src/components/authenticator.vue

```diff
@@ -407,4 +412,24 @@ const onConfirmVerifyUserSubmitI = (e: Event) => {
     confirmVerifyUserComponent.value.submit(e);
   }
 };
+
+/**
+ * Toggle sign up and sign in pages when useSelect
+ * active ref updates
+ */
+
+watch(
+  () => active.value,
+  () => {
+    if (active.value) {
+      send({
+        type: 'SIGN_UP',
+      });
+    } else {
+      send({
+        type: 'SIGN_IN',
+      });
+    }
+  }
+);
```

Comment:
This ties into the simple `base-two-tabs` component. I use this to toggle which pages to show.